packages/psnotify.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,17 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+    $toolName = 'psnotify'
+    $category = 'dotNet'
+
+    $zipUrl = 'https://github.com/WithSecureLabs/GarbageMan/releases/download/v0.2.4/psnotify.zip'
+    $zipSha256 = '255633da6e61bf30a67bce995ef72b7f9d8c85c75c8c5ee0aedb48709f7e6454'
+
+    $executablePath = (VM-Install-From-Zip $toolName $category $zipUrl -zipSha256 $zipSha256 -innerFolder $true)
+    $innerPath = Join-Path $executablePath[0] "psnotify" -Resolve
+    Move-Item -Path $innerPath -Destination "C:\" -Force -ea 0
```

Comment:
Why do you need to move psnotify to `C:\`?

Reply:
@Ana06 You are correct, this does currently break the shortcut and bin file that are created. So you suggestion might be a good solution as we need to be able to adjust the $toolDir path, which defaults to `_Install-BinFile_` at the start of the function.